### DISCUSSION TITLE
Plex Player Support

### DISCUSSION THREAD

**Discussion by Deathbot64** [music-assistant/support#2227] [Feature requests and ideas]
Being able to send music to plexamp or plex apps like the plex integration in HA would make this perfect.

**Comment by jannis3005** [comment]
+1 Would be very cool :)

**Comment by OzGav** [comment]
Current information is that it is not possible to send to Plexamp as Plexamp headless devices only support Plex as a source

https://discord.com/channels/753947050995089438/1377149388652285962/1379300220721958972

**Comment by OzGav** [comment]
Plex Connect support is available on 2.7.0 beta 16 and later

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
